src/mmw/js/src/core/views.js

```diff
@@ -59,6 +61,10 @@ var HeaderView = Marionette.ItemView.extend({
         // Defer requiring app until needed as it is not defined when
         // core.views are initialized (they are required in app.js)
         require('../app').showLoginModal();
+    },
+
+    userLogout: function() {
+        require('../app').userLogout();
```

Comment:
Actually, I think we just need to pass a reference to App in the constructor of this view. It's not a best practice to have require statements at "run-time" like this.